tests/unit/commands/__init__.py

```diff
@@ -43,8 +45,13 @@ def get_sample_key(name):
     raise KeyError(name)
 
 
-def mock_snap_output(command, *args, **kwargs):
-    if command == ["snap", "keys", "--json"]:
+original_check_output = subprocess.check_output
```

Comment:
This could use an explanation :)

Reply:
Sure, it is a pattern we have been using across our tests, but I can certainly doc it